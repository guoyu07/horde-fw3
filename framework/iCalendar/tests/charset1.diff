002- string(6) "möchen"
002+ string(7) "mÃ¶chen"