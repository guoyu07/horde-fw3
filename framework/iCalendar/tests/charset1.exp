string(6) "möchen"
string(6) "möchen"