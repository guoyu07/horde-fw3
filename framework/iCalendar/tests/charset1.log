
---- EXPECTED OUTPUT
string(6) "möchen"
string(6) "möchen"
---- ACTUAL OUTPUT
string(6) "möchen"
string(7) "mÃ¶chen"
---- FAILED
