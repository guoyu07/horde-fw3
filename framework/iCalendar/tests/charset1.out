string(6) "möchen"
string(7) "mÃ¶chen"